src/Admin/AdminInterface.php

```diff
@@ -433,49 +336,39 @@ public function hasSubject();
      * @deprecated this feature cannot be stable, use a custom validator,
      *             the feature will be removed with Symfony 2.2
      */
-    public function validate(ErrorElement $errorElement, $object);
+    public function validate(ErrorElement $errorElement, $object): void;
```

Comment:
```suggestion
    public function validate(ErrorElement $errorElement, object $object): void;
```

Reply:
This method should be removed as per "NEXT_MAJOR" comment.